src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,108 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh):
+  """
+  @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information
+  @param input_mesh path of the input abaqus file
+  @param output_mesh path of the output gmsh file
+  """
+  # Load the mesh
+  print('Reading abaqus mesh...')
+  mesh = meshio.Mesh.read(input_mesh, "abaqus")
+
+  # Add a default region tag
+  print('Converting region tags...')
+  region_list = list(mesh.cell_sets.keys())
+  Nregion = len(region_list)
+  cell_ids = np.zeros(len(mesh.cells[0][1]), dtype=int)
```

Comment:
Is `0` meant to be the default value you want when you do not put any value?
Are all the initial values overwritten?
Consider using `numpy.empty` instead: if there is an error, you'll see it fast with crazy dumb values.

Reply:
@TotoGaz - gmsh interprets a zero value as a missing tag, so that was the motivation here.  But, I can see the benefit for easier error-checking...